[TH]_Psycho_Pass_THE_MOVIE.ass

```diff
@@ -24,84 +26,84 @@ Style: TitleCard,Open Sans Semibold,63,&H00FFFFFF,&H000000FF,&H00020713,&H000000
-Dialogue: 0,0:06:34.93,0:06:38.27,Default,,0,0,0,,Inspector Shimozuki will take the elevator with\NSugou and Hinakawa,
-Dialogue: 0,0:06:38.44,0:06:41.23,Default,,0,0,0,,putting two drones in the lead.
-Dialogue: 0,0:06:42.19,0:06:43.23,Default,,0,0,0,,Roger that.
-Dialogue: 0,0:06:44.11,0:06:45.94,Default,,0,0,0,,First, my team will enter
-Dialogue: 0,0:06:46.03,0:06:48.45,Default,,0,0,0,,using flashbang grenades,
-Dialogue: 0,0:06:48.91,0:06:50.95,Default,,0,0,0,,which will distract the enemy.
-Dialogue: 0,0:06:51.28,0:06:53.45,Default,,0,0,0,,At that time, Inspector Shimotsuki's team
-Dialogue: 0,0:06:53.58,0:06:56.75,Default,,0,0,0,,will enter from the rear and flank the enemy,
-Dialogue: 0,0:06:56.83,0:06:58.46,Default,,0,0,0,,trapping all of them.
-Dialogue: 0,0:07:00.04,0:07:01.42,Default,,0,0,0,,Flashbangs?
-Dialogue: 0,0:07:01.92,0:07:03.88,Default,,0,0,0,,You're being really careful.
-Dialogue: 0,0:07:04.13,0:07:06.30,Default,,0,0,0,,If the information from the Pillbugs is correct,
-Dialogue: 0,0:07:06.46,0:07:08.59,Default,,0,0,0,,the enemy is equipped with firearms.
-Dialogue: 0,0:07:08.93,0:07:10.09,Default,,0,0,0,,You mean...
-Dialogue: 0,0:07:10.64,0:07:15.10,Default,,0,0,0,,If we allow the enemy to counterattack,\Nthere will likely be casualties.
-Dialogue: 0,0:07:15.43,0:07:17.43,Default,,0,0,0,,Take extreme caution.
-Dialogue: 0,0:07:18.02,0:07:18.94,Default,,0,0,0,,Roger.
+Dialogue: 0,0:00:31.95,0:00:36.74,Default,,0,0,0,,ราวกับว่าพวกเราเป็นสิ่งมีชีวิตที่กำลังวิวัฒนาการอยู่บนจุดสูงสุด
+Dialogue: 0,0:00:36.87,0:00:40.41,Default,,0,0,0,,บางครั้งสูงยิ่งกว่ายอดหอระฆังของโบสถ์
+Dialogue: 0,0:00:40.58,0:00:45.42,Default,,0,0,0,,ในที่สุด การพัฒนาต่อไปนั้นกลายเป็นเรื่องยากและอันตราย
+Dialogue: 0,0:00:45.58,0:00:47.92,Default,,0,0,0,,สิ่งที่รอคอยก็คือการล่มสลาย
+Dialogue: 0,0:00:49.38,0:00:50.59,Default,,0,0,0,,พวกมันกำลังมา
+Dialogue: 0,0:00:56.43,0:00:57.68,Default,,0,0,0,,ซ้าย 3 คลิก
+Dialogue: 0,0:00:58.76,0:01:00.27,Default,,0,0,0,,ขวา 4 คลิก
+Dialogue: 0,0:01:01.39,0:01:04.60,Default,,0,0,0,,ซ้าย 1 คลิก\Nขวา 2 คลิก
+Dialogue: 0,0:01:07.23,0:01:07.94,Default,,0,0,0,,ตอนนี้แหละ
+Dialogue: 0,0:01:09.94,0:01:11.61,Default,,0,0,0,,เยี่ยมมาก โควกามิ
+Dialogue: 0,0:01:21.74,0:01:25.58,Default,,0,0,0,,เราจะบอกว่ามันมีกฏของศิลปะที่โหดร้ายอยู่\N(ผู้แต่งบทกวีนี้กำลังพูดถึงเรื่องช่วงเวลาชีวิตของคน กับสิ่งที่มีค่า)\N (ตอนทำซับตรงนี้น่าจะใส่คำอธิบายหน่อยว่าเป็นบทกวี)
+Dialogue: 0,0:01:25.79,0:01:27.92,Default,,0,0,0,,ก่อนอื่น มนุษย์ทุกคนควรจะตายจากโลกนี้ไปให้หมด\N(ก่อนที่จะสร้างผลงานที่มีคุณค่าได้)
+Dialogue: 0,0:01:28.21,0:01:31.25,Default,,0,0,0,,แล้วก็พวกเราจะตายตามไป\N(มีค่าพอที่โลกจะจดจำก่อนจากโลกนี้ไป)
+Dialogue: 0,0:01:31.42,0:01:34.76,Default,,0,0,0,,หลังจากทนทุกข์ทรมานอย่างสาสม\N(จะต้องผ่านความยากลำบากมานับไม่ถ้วน)
+Dialogue: 0,0:01:35.13,0:01:39.60,Default,,0,0,0,,ไม่ใช่หญ้าแห่งการลืมเลือน แต่เป็นหญ้าอันเป็นนิรันด์\N(เพื่อสร้างผลงานที่เลื่องชื่อ ไม่ใช่ผลงานที่ถูกลืม)
```

Comment:
จากที่ฟังเสียง eng คิดว่าซับน่าจะผิดครับ น่าจะไม่ได้ขึ้นต้นด้วยคำว่า so that (เพื่อว่า) นะครับ คิดว่านะจะเป็น
"As the grass, not the oblivion, but the eternal life" / "That the grass, not the oblivion, but the eternal life" แต่ถ้าประโยคเป็นแบบนี้ก็แปลไม่ถูกนี่สิครับ (ก็ไม่รู้ว่าฟังถูกรึเปล่านะครับ)